src/catalog/database.py

```diff
@@ -0,0 +1,111 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, scoped_session
+from src.configuration.dictionary import SQLALCHEMY_DATABASE_URI
+from sqlalchemy.ext.declarative import declarative_base, declared_attr
+from sqlalchemy.exc import DatabaseError
+from sqlalchemy_utils import database_exists, create_database, drop_database
+
+engine = create_engine(SQLALCHEMY_DATABASE_URI)
```

Comment:
Should these variables be global?

Can we move the `database uri` to `configuration.yml`?